tests/integration/spec.bats

```diff
@@ -66,12 +66,18 @@ function teardown() {
 }
 
 @test "spec validator" {
+  TESTDIR=$(pwd)
   cd "$HELLO_BUNDLE"
-  # note this test runs from the temporary bundle directory not the integration root
-  # note this test is brittle against specs changes that lead runc's spec command
-  # todo get the validate program, gojsonschema, and schema/*s.json from godeps?
 
   run git clone https://github.com/opencontainers/runtime-spec.git src/runtime-spec
+  [ "$status" -eq 0 ]
+
+  SPEC_COMMIT=$(grep runtime-spec ${TESTDIR}/../../Godeps/Godeps.json -A 4 | grep Rev | cut -d":" -f 2 | tr -d ' "')
+  cd src/runtime-spec
+  run git reset --hard $SPEC_COMMIT
+  [ "$status" -eq 0 ]
+
+  cd "$HELLO_BUNDLE"
```

Comment:
Instead of `cd`ing in and back, you can use something like `run git -C src/runtime-spec reset --hard "${SPEC_COMMIT}"`.